.github/workflows/compile-docker.yml

```diff
@@ -7,13 +7,12 @@ jobs:
     strategy:
       matrix:
         include:
-          - board: "raspberry-pi"
-            arch: "arm"
```

Comment:
also imo we should leave armv6 in. ideally we should support it in future and since it works here i dont see the reason to remove it right now

Reply:
I really don't understand your arm v6 passion :smile: I'm leaning to cut our build times and keep things cleaner. I don't think we'll ever use arm v6 for any of our devices. Also we've removed arm v6 from other containers. Why to keep for this one? Unless of course you have a particular reason.